indigo_app/static/stylesheets/_akn.scss

```diff
@@ -74,7 +76,9 @@ $hierarchy-elems: (
 
 // Position hierarchical and block AKN elements relatively,
 // so that inserted indicators and annotations can be positioned relatively
-.akoma-ntoso {
+.akoma-ntoso,
```

Comment:
This `.akoma-ntoso` can fall away. We only do this in places where we use `la-akoma-ntoso`.